lighting_import_attachment_product_queue_job/models/lighting_import_attachment_file.py

```diff
@@ -0,0 +1,58 @@
+# Copyright NuoBiT Solutions - Frank Cespedes <fcespedes@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import _, api, fields, models
+
+
+class LightingImportAttachmentFile(models.Model):
+    _inherit = "lighting.import.attachment.file"
+
+    file_jobs_ids = fields.Many2many(
+        comodel_name="queue.job",
+        relation="lighting_import_attachment_file_queue_job_rel",
+        column1="file_id",
+        column2="job_id",
+        string="Queue Jobs",
+        copy=False,
+    )
+    file_job_error = fields.Boolean(compute="_compute_file_job_error")
+
+    @api.depends("file_jobs_ids", "file_jobs_ids.state")
+    def _compute_file_job_error(self):
+        for rec in self:
+            rec.file_job_error = bool(
+                rec.file_jobs_ids.filtered(lambda r: r.state == "failed")
```

Comment:
What if the state is "pending" or "enqueued" or others, it would seem the file is correctly processed but it's not, maybe using another color when it's processing? Or maybe painting the line green??

Also, the 'imported' status should only be applied when all files are processed correctly. Right now it changes to "imported" just after the jobs are created and instead it should be after all jobs were executed correctly. The process is asynchronous so the other things like this should be changed accordingly.